### PR TITLE
[O11y][Azure] Remove suffix from Compute VMs Overview dashboard

### LINKED ISSUE #7585: [O11y][Azure] Remove `[Azure Metrics]` suffix to the panels titles
## Description
The `VM Disk Reads [Azure Metrics]` is the only panel having the `[Azure Metrics]` suffix from Compute VMs Overview dashboard.
According to the best practices "[Azure Metrics]" suffix should be added to the all panel titles.

### Screenshots
image

### PR DESCRIPTION
**Pull request by harnish-elastic**
- Enhancement

## Proposed commit message

- According to the [issue](https://github.com/elastic/integrations/issues/7585), need to remove the suffix `[Azure Metrics]` in two panels which becomes inconsistent looking at the other panel titles.
- Removed from Markdown panel → Navigation Compute VM Overview
- Removed from visualization → VM Disk Reads

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes #7585

#### Before

image

#### After

image

### CODE REVIEW

**Review by kush-elastic — APPROVED** [review]
LGTM!

File: packages/azure_metrics/kibana/dashboard/azure_metrics-eb3f05f0-ea9a-11e9-90ec-112a988266d5.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
                             "markdown": "### Azure Compute VM \n\n[**Overview**](#/dashboard/azure_metrics-1a151f80-32db-11ea-a83e-25b8612d00cc) | [Guest Windows VM Metrics](#/dashboard/azure_metrics-b6b434c0-feae-11eb-8fff-87871c582eca) | [Guest Linux VM Metrics](#/dashboard/azure_metrics-d8e3c5f0-feaf-11eb-8fff-87871c582eca)",
                             "openLinksInNewTab": false
                         },
-                        "title": "Navigation Compute VM Overview [Azure Metrics]",
+                        "title": "Navigation Compute VM Overview",
```
Comment (muthu-mps) [inline packages/azure_metrics/kibana/dashboard/azure_metrics-eb3f05f0-ea9a-11e9-90ec-112a988266d5.json:45]: While performing this change we may need to update the integrations which follows similar patterns,

Screenshot 2024-04-02 at 10 34 33 AM

#### Integrations with similar naming pattern

- Storage Metrics
- Container Instance
- Container Registry and
- Container Service

#### Similar Issue in Other Azure Metrics Integrations
Performed a random check and in the other metrics integrations dashboard and there are similar issue that needs to be addressed. Please find the dashboard screenshot below,

Screenshot 2024-04-02 at 10 45 10 AM

Lets create a separate PR to address this issue.

**Review by muthu-mps — APPROVED** [review]
Left a comment to address the similar label issue in other Azure Metrics integrations. Otherwise change looks good!

**Review by devamanv — APPROVED** [review]
LGTM

**Review by zmoog — APPROVED** [review]
LGTM

**Pull request merged**